<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1920</width>
    <height>1080</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>1920</width>
    <height>1080</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QWidget" name="verticalLayoutWidget">
    <property name="geometry">
     <rect>
      <x>1200</x>
      <y>30</y>
      <width>435</width>
      <height>116</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QLabel" name="bonusText">
       <property name="text">
        <string>Получайте очки за каждого нарушителя правил расхождения</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="scoreLabel">
       <property name="font">
        <font>
         <pointsize>20</pointsize>
        </font>
       </property>
       <property name="text">
        <string>Счёт:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="scoreLabel2">
       <property name="font">
        <font>
         <pointsize>20</pointsize>
         <italic>true</italic>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="loseLabel">
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QPushButton" name="startBttn">
    <property name="geometry">
     <rect>
      <x>386</x>
      <y>847</y>
      <width>811</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>Начать игру</string>
    </property>
   </widget>
   <widget class="QGraphicsView" name="graphicsView">
    <property name="geometry">
     <rect>
      <x>386</x>
      <y>31</y>
      <width>810</width>
      <height>810</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>810</width>
      <height>810</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>820</width>
      <height>820</height>
     </size>
    </property>
   </widget>
   <widget class="QLabel" name="textLabel">
    <property name="geometry">
     <rect>
      <x>710</x>
      <y>340</y>
      <width>581</width>
      <height>101</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Ubuntu Mono</family>
      <pointsize>20</pointsize>
     </font>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>70</y>
      <width>361</width>
      <height>111</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>15</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Здесь вы можете начать игру </string>
    </property>
   </widget>
   <widget class="Line" name="line">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>10</y>
      <width>20</width>
      <height>101</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Vertical</enum>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1920</width>
     <height>22</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuPause">
    <property name="title">
     <string>Пауза</string>
    </property>
    <addaction name="action"/>
    <addaction name="separator"/>
    <addaction name="action_3"/>
   </widget>
   <addaction name="menuPause"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="action">
   <property name="text">
    <string>Новая игра</string>
   </property>
  </action>
  <action name="action_3">
   <property name="text">
    <string>Обучение</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
